### DISCUSSION TITLE
Search engine with Directus Cloud hosted

### DISCUSSION THREAD

**Discussion by Leotheluck** [directus/directus#15544] [Archive]
Hello everyone,

I'm currently working on a project which is a news website in Next.js with a back-end hosted on Directus Cloud, I'm trying to implement a search engine and I can only guess doing "search=" requests to the API isn't the way to go 😅 Would you know how I could implement Elastic or Algolia with Directus Cloud?

Thanks a lot!

**Comment by rijkvanzanten** [comment]
You could have Algolia scrape the output of the Directus API, but if it's for website search specifically, you might want to give Algolia's Crawler + Autocomplete combo a shot 🙂 I started using that after trying it through DocSearch and am a big fan
